projects/control-service/projects/helm_charts/pipelines-control-service/templates/_helpers.tpl

```diff
@@ -244,6 +267,10 @@ Image Pull Secret in json format
     {{ include "buildImagePullSecretJson" (list .Values.image.registry .Values.image.registryUsernameReadOnly .Values.image.registryPasswordReadOnly) }}
 {{- end }}
 
+{{- define "operationsUiPullSecretJson" }}
+    {{ include "buildImagePullSecretJson" (list .Values.operationsUi.registry .Values.operationsUi.username .Values.operationsUi.password) }}
+{{- end }}
+
```

Comment:
I find those configs like `.Values.image.registryUsernameReadOnly` to be more understandable (due username or password are not suggesting what those are about)

Reply:
@ivakoleva I have now updated it to be `image.username`. Do you think that is descriptive enough? 
 
In fact I think my one is more descriptive. 

`.Values.image.registryUsernameReadOnly` it is unclear what image this username would be pulling. 


Whereas with my it is obvious that the username is for the image operationsUi. 

Mine is also more consistent with the rest of the image usernames and passwords. e.g 


```
{{ include "buildImagePullSecretJson" (list .Values.deploymentBuilderImage.registry .Values.deploymentBuilderImage.username .Values.deploymentBuilderImage.password) }}
```